src/js/enableChromecast.js

```diff
@@ -41,8 +42,8 @@ function onChromecastRequested(player) {
 }
 
 /**
- * Adds the Chromecast button to the player's control bar, if one does not already exist,
- * then starts listening for the `chromecastRequested` event.
+ * Adds the Chromecast button to the player's control bar, if one does not already
+ * exist, then starts listening for the `chromecastRequested` event
```

Comment:
This change seems superfluous?

Reply:
https://github.com/silvermine/videojs-chromecast/pull/112/commits/fef37e6fe2f4a76abb024b277cc4bbf5fcf65524